src/tutorial/src/step-4/description.md

```diff
@@ -55,15 +55,15 @@ createApp({
 
 </div>
 
-Inside a method, we can access the component instance using `this`. The component instance exposes the data properties declared by `data`. We can update the component state by mutating these properties.
+All'interno di un metodo, possiamo accedere all'istanza del componente utilizzando `this`, il quale espone le proprietà dichiarate in `data` e possiamo aggiornare lo stato del componente modificando queste proprietà.
```

Comment:
qui ti direi di mantenere le frase separate, capisco però che è più scorrevole.

Suggerimento:
->
All'interno di un metodo, possiamo accedere all'istanza del componente utilizzando `this`. L'istanza, poi, espone le proprietà dichiarate in `data`. In questo modo possiamo aggiornare lo stato del componente modificando queste proprietà.